shopfloor/services/zone_picking.py

```diff
@@ -664,13 +703,20 @@ def _scan_source_product(
             )
         else:
             response = self._list_move_lines(
-                sublocation or self.zone_location, sublocation=sublocation
+                sublocation or self.zone_location,
+                sublocation=sublocation,
```

Comment:
regarding my previous remark, this would be sufficient
```suggestion
                sublocation or self.zone_location,
```